devito/compiler.py

```diff
@@ -448,8 +448,7 @@ def __init__(self, *args, **kwargs):
         self.cflags.remove('-O3')
         self.cflags.remove('-Wall')
         self.cflags += ['-fast', '-acc']
-        # self.cflags += ['-Minfo=accel'] # Print informational messages for compliation
-        # self.cflags += ['-ta=tesla']    # Compile for a target GPU architecture
+        #self.cflags += ['-ta=tesla,host']    # Default PGI compile for a target is GPU and single threaded host.
```

Comment:
@kenhester thanks to #1383 (from our summer student @jack-lascala) we'll soon be able to auto-detect whether it's an actual tesla card what we got underneath !